_includes/challenge_card.html

```diff
@@ -45,6 +45,28 @@
             </li>
         {% endif %}
 
+        {% if challenge.file_path %}
+             <li class="list-group-item">
+                <div class="form-group row">
+                    <div class="col-sm-2">
+                        <strong>Hints </strong>
+                    </div>
+                    <div class="col-sm-5">
+                        <details>
+                            <summary>Click to expand</summary>
+                            {% if challenge.hints %}
+                                {% for hint in challenge.hints %}
+                                    {{ hint }}
```

Comment:
Still will just show all the hints on the same line, unless you wrap them in a <li> or something